src/services/codefixes/helpers.ts

```diff
@@ -945,6 +945,19 @@ function findJsonProperty(obj: ObjectLiteralExpression, name: string): PropertyA
     return find(obj.properties, (p): p is PropertyAssignment => isPropertyAssignment(p) && !!p.name && isStringLiteral(p.name) && p.name.text === name);
 }
 
+/**
+ * @internal
+ * Information about an `import("...").SomeType` replaced by an auto-importable type reference.
+ */
+export interface ImportTypeReplacement {

```

Comment:
"`replacement`" is a little confusing to me--does this refer to export aliases in general?

Reply:
No, it refers to the fact that an ImportType has been replaced by a TypeReference plus a top-level import.